http/query_handler.go

```diff
@@ -208,6 +214,32 @@ func (h *FluxHandler) handleQuery(w http.ResponseWriter, r *http.Request) {
 			zap.Error(err),
 		)
 	}
+
+	// Detailed logging for flux queries if enabled
+	if h.FluxLogEnabled {
+		c, ok := req.Request.Compiler.(lang.FluxCompiler)
+		if !ok {
+			h.log.Error("failed to log flux query")
+			return
+		}
+
+		h.logFluxQuery(cw.Count(), stats, c, err)
+	}
+
+}
+
+func (h *FluxHandler) logFluxQuery(n int64, stats flux.Statistics, compiler lang.FluxCompiler, err error) {
```

Comment:
Very nearly a 1:1 copy of the equivalent 1.x code, see: https://github.com/influxdata/influxdb/blob/444c22b67d425dbd23d8b71690effcfe8cbcba58/services/httpd/handler.go#L1442